src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -115,6 +116,86 @@ void initService() {
         config);
   }
 
+  @Test
+  void sendMessage() throws Exception {
```

Comment:
This isn't specific to this PR, it's a more detailed test for the common `sendMessage()` code so it doesn't need to be tested in each case.